yt_dlp/extractor/zee5.py

```diff
@@ -97,14 +97,14 @@ def _real_extract(self, url):
             (lambda x: x['hls'][0], lambda x: x['video_details']['hls_url']),
             str)
         formats = self._extract_m3u8_formats(
-            'https://zee5vodnd.akamaized.net' + m3u8_url.replace('/drm1/', '/hls1/') + token_request['video_token'],
+            'https://zee5vodnd.akamaized.net' + m3u8_url.replace('drm', 'hls') + token_request['video_token'],
             video_id, fatal=False)
         mpd_url = try_get(
             json_data,
             (lambda x: x['video'][0], lambda x: x['video_details']['url']),
             str)
         formats += self._extract_mpd_formats(
-            'https://zee5vodnd.akamaized.net' + mpd_url + token_request['video_token'],
+            'https://zee5vod.akamaized.net' + mpd_url,
```

Comment:
why? It was working before, correct?

Reply:
Also see https://github.com/yt-dlp/yt-dlp/pull/278#issuecomment-827805138

Reply:
If you use `--allow-unplayable-formats` this link will not work it will give you 403 error.

try this command `yt-dlp --allow-unplayable-formats https://www.zee5.com/tvshows/details/kundali-bhagya/0-6-366/kundali-bhagya-april-28-2021/0-1-manual-bg3l526se9e0` it will show 403 error

Reply:
Are there never any non-DRM formats in the mpd?

Reply:
No.. Zee5 uses DRM in all mpd links I have researched a lot on zee5.